recipes/fparser/LICENSE

```diff
@@ -0,0 +1,63 @@
+Modified work Copyright (c) 2017 Science and Technology Facilities Council
```

Comment:
Looks like the license file is available in the pypi tarball, so please delete the file from here:
```
fparser-0.0.10/LICENSE
x fparser-0.0.10/doc/
x fparser-0.0.10/doc/conf.py
x fparser-0.0.10/doc/Makefile
x fparser-0.0.10/doc/introduction.rst
```